speakers/index.html

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Give a talk!
+description: "Interested in giving a talk at Amsterdam.rb? Read this page for all the information you need to prepare and what you can expect."
+---
+
+<main class="light-content content-with-shadow">
+  <div class="wrap text-content">
+    <h1>Give A <span class="highlight">Talk</span></h1>
+    <p>
+      We're always looking for new and experienced speakers to give a talk at our meetup.
+      First time speaker are very welcome to send in their talks.
+    </p>
+    <p>
+      If you are interested in giving a talk at one of our future meetups, awesome!, contact us at <a href="mailto:contact@rubynl.org?subject=Giving%20a%20talk%20at%Amsterdam%20Ruby">contact@rubynl.org</a> with a talk title and a short description. You can send in your talk idea months in advance, even before you have starting working on the slides.
```

Comment:
Suggestion: If you are interested in giving a talk at one of our future meetups, awesome! Contact us at ...

Reply:
I'll add a message that this can be found at the top of the page. I don't want to repeat that content again.